samples/apps/bookinfo/CONSUL.md

```diff
@@ -0,0 +1,76 @@
+# Consul Adapter for Istio on Docker
+
+Make Istio run in docker environment by integrating Consul as a service registry.
+
+## Design Principle
+
+The key issue is how to implement the ServiceDiscovery interface functions in Istio. 
+This platform adapter uses Consul Server to help Istio monitor service instances running in the underlying platform.
+When a service instance is brought up in docker, the [Registrator](http://gliderlabs.github.io/registrator/latest/)
+automatically registers the service in Consul.
+
+Noted that Istio pilot is running inside each app container so as to coordinate Envoy and the service mesh.
```

Comment:
This is about pilot agent, isn't it? We refer to Istio pilot as the central pilot, taking care of discovery.